### ISSUE TITLE
Broken Sketch State - Parallel Command

### ISSUE THREAD

**Issue by jgomez720** [KittyCAD/modeling-app#4164]
### Description
When doing a parallel constraint on the last line (which is probably the problem, tbh.), the sketch state breaks. I can't escape, can't draw anything new, etc. I can only refresh

### KCL
```js
length001 = 10
length002 = 4
sketch001 = startSketchOn('XY')
  |> startProfileAt([-7.39, 3.99], %)
  |> angledLine([0, length001], %)
  |> angledLine([segAng(seg01) + 180, length002], %)
  |> line([-14.28, 0.07], %)
  |> lineTo([profileStartX(%), profileStartY(%)], %, $seg01)
  |> close(%)
```

### Version
v0.25.6

### Recordings/Screenshots
sorry, didn't mute audio
https://github.com/user-attachments/assets/85e2949a-4aac-4978-be36-58bcf291c1fc

**Comment by lf94** [comment]
@franknoirot this is fixed right, just not released?

**Comment by franknoirot** [comment]
Yes @lf94 thank you. @jgomez720 this should have been fixed by #4138, which was released to the desktop app with #4196. I'm going to close this as completed, but please reopen if you see otherwise.
